tests/unittests/server/utils/jsonpatch/test_jsonpatch.py

```
@@ -173,3 +173,98 @@ def test_ok_multi(transform_fn, patches):
                     if "value" in patch
                 ]
             )
+
+
+class TestIsRootDelete:
+    """Tests for jsonpatch.is_root_delete utility function."""
+
+    @pytest.mark.parametrize(
+        "operations,expected",
```

Ruff (0.14.13)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)